src/pages/Login/Login.test.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { shallow } from "enzyme";
+import Login from "./Login";
+
+jest.mock("../../context/AuthContext", () => ({
+  useAuthContext: () => ({
+    signin: jest.fn(),
+  }),
+}));
+
+const mockIsAuthenticated = false;
+
+jest.mock("../../hooks/useProvideAuth", () => ({
+  useProvideAuth: () => ({
+    isAuthenticated: mockIsAuthenticated,
+  }),
+}));
+
+describe("Login", () => {
+  const setup = () => shallow(<Login />);
+
+  const findByTestAttribute = (wrapper, val) =>
+    wrapper.find(`[data-test='${val}']`);
+
+  test("renders without error", () => {
+    const wrapper = setup();
+    expect(wrapper.exists()).toBeTruthy();
+  });
+
+  test("renders login display", () => {
+    const wrapper = setup();
+    const loginDisplay = findByTestAttribute(wrapper, "login-display");
+    expect(loginDisplay.length).toBe(1);
+  });
+
+  test("renders button display", () => {
+    const wrapper = setup();
+    const loginButtonDisplay = findByTestAttribute(wrapper, "login-button");
+    expect(loginButtonDisplay.length).toBe(1);
+  });
+
+  test("button is clicked", () => {
+    const wrapper = setup();
+    const button = findByTestAttribute(wrapper, "login-button");
+    button.simulate("click");
+    expect(button.length).toBe(1);
```

Comment:
🔧 I think we need to check that the button has been clicked, not that there is a button here.

Reply:
Testing button click completed